apps/nextjs/src/app/[locale]/manage/medias/_actions/copy-media.tsx

```diff
@@ -15,7 +15,7 @@ export const CopyMedia = ({ media }: CopyMediaProps) => {
 
   const url =
     typeof window !== "undefined"
-      ? `${window.location.protocol}://${window.location.hostname}:${window.location.port}/api/user-medias/${media.id}`
+      ? `${window.location.protocol}//${window.location.hostname}:${window.location.port}/api/user-medias/${media.id}`
```

Comment:
I think we can fix another issue here when we remove the port thing when it's not set (empty string)